src/OrchardCore.Themes/TheAdmin/wwwroot/Scripts/TheAdmin.js

```diff
@@ -7099,24 +7099,25 @@ $(function () {
 
 $(function () {
```

Comment:
I'm curious, why wrap the function in $() ?

Reply:
Me too :)